script/check-simulation-build-cmake

```diff
@@ -40,31 +40,31 @@ build_all_features()
 {
     # Build Thread 1.1 with full features
     reset_source
-    "$(dirname "$0")"/cmake-build simulation
+    "$(dirname "$0")"/cmake-build simulation -DOT_THREAD_VERSION=1.1
 
     # Build Thread 1.2 with full features
     reset_source
-    "$(dirname "$0")"/cmake-build simulation -DOT_OTNS=ON -DOT_SIMULATION_VIRTUAL_TIME=ON -DOT_THREAD_VERSION=1.2 -DOT_DUA=ON -DOT_MLR=ON -DOT_BACKBONE_ROUTER=ON -DOT_CSL_RECEIVER=ON
+    "$(dirname "$0")"/cmake-build simulation -DOT_OTNS=ON -DOT_SIMULATION_VIRTUAL_TIME=ON -DOT_DUA=ON -DOT_MLR=ON -DOT_BACKBONE_ROUTER=ON -DOT_CSL_RECEIVER=ON
```

Comment:
How about keeping the `OT_THREAD_VERSION=1.2` option?

Reply:
Sounds good :+1:  Updated.